include/deal.II/fe/fe_series.h

```diff
@@ -435,6 +471,28 @@ FESeries::process_coefficients(
 }
 
 
+
+template <int dim, int spacedim>
+template <class Archive>
+inline void
+FESeries::Fourier<dim, spacedim>::serialize(Archive &ar,
+                                            const unsigned int /*version*/)
+{
+  ar &fourier_transform_matrices;
```

Comment:
is this the only member function of this class? In general, after deserialization you should have the exact same member function again, and it should compare equal.

Reply:
This class stores `SmartPointer` objects to a `FECollection` and `QCollection` objects. I don't know how to deal with them.

There are also a container for all `k_vectors` in the Fourier class and one integer `N` in the Legendre class which will be set upon construction. I don't know to serialize them either. Shall I simply override those created on construction?

Reply:
As discussed, let's just save/restore the matrices and reflect that in the function names.

Reply:
I'm preparing corresponding functions now. I've found a good basis for them in `grid/tria.h`, i.e. both functions `Triangulation::{save|load}()`.

However, to make this fail-proof, I'll need a `get_name()` function for the `Quadrature` classes as well, similar to the `FiniteElement` ones. I'll provide them in a separate PR. Do you agree, or do you have a different idea?

Reply:
That's one way. Alternatively, you could just output the quadrature points and weights as well, and upon reading stuff back in verify that they're still the same.

Reply:
An alternative would be to compute a has of the points and weights. `std::hash` could be overloaded for `Quadrature<dim>` (https://en.cppreference.com/w/cpp/utility/hash) or you could just implement a `Quadrature::hash()` function.